### PR TITLE
test: repair test

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve? 

repair test failed in TestStoreOverloaded and TestStoreOverloadedWithReplace with deadlock-enable.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: server/coordinator_test.go [hunk 1]
```diff
@@ -801,22 +803,28 @@ func (s *testOperatorControllerSuite) TestStoreOverloaded(c *C) {
...
 	c.Assert(oc.AddOperator(op1), IsTrue)
 	for i := 0; i < 10; i++ {
+		if time.Now().Sub(start) >= time.Second*9/10 {
+			break
+		}
 		c.Assert(lb.Schedule(tc), IsNil)
 	}
 	c.Assert(oc.RemoveOperator(op1), IsTrue)
-	time.Sleep(1 * time.Second)
+	time.Sleep(time.Duration(60/cfg.StoreBalanceRate) * time.Second)
```
Comment (nolouch) [inline server/coordinator_test.go]: It's too long to wait for `6s`, maybe 2 s?

Comment (lhy1024) [reply server/coordinator_test.go]: I have tried 2s,3s,5s, but failed.

Comment (nolouch) [reply server/coordinator_test.go]: why failed? In theory, only one second is enought.

Comment (lhy1024) [reply server/coordinator_test.go]: Maybe because I used `60/cfg.StoreBalanceRate` to adjust the sleep time, it is now changed to 2 seconds.

**Comment by shafreeck** [comment]
What is the relation between these test cases with `deadlock-enable`?

**Comment by lhy1024** [comment]
> What is the relation between these test cases with `deadlock-enable`?

Deadlock detection causes the token to not be acquired properly on a high-load machine, so the test fails

**Pull request merged**